pkg/bgpv1/manager/route_policy_reconciler.go

```diff
@@ -31,16 +32,21 @@ type RoutePolicyReconcilerOut struct {
 }
 
 type RoutePolicyReconciler struct {
-	ipPoolStore BGPCPResourceStore[*v2alpha1api.CiliumLoadBalancerIPPool]
+	lbStore  BGPCPResourceStore[*v2alpha1api.CiliumLoadBalancerIPPool]
+	podStore BGPCPResourceStore[*v2alpha1api.CiliumPodIPPool]
```

Comment:
What about naming these `lbPoolStore` and `podPoolStore`?

Reply:
Commit 2085898 resolves this conversation.